dcat/rdf/dcat3.ttl

```diff
@@ -335,6 +338,32 @@ dcat:Dataset
   skos:scopeNote "Denne klasse repræsenterer det konkrete datasæt som det udgives af datasætleverandøren. I de tilfælde hvor det er nødvendigt at skelne mellem det konkrete datasæt og dets registrering i kataloget (fordi metadata såsom ændringsdato og vedligeholder er forskellige), så kan klassen katalogpost anvendes. "@da ;
   skos:scopeNote "Denne klasse beskriver det konceptuelle datasæt. En eller flere repræsentationer kan være tilgængelige med forskellige skematiske opsætninger, formater eller serialiseringer."@da ;
 .
+dcat:DatasetSeries
+  a rdfs:Class ;
+  a owl:Class ;  
+  rdfs:comment "A collection of datasets that are published separately, but share some common characteristics that groups them."@en ;
```

Comment:
share + common is tautology. 
Either 'have some common characteristics' or 'share some characteristics'. 

Also groups --> group (english grammar)

Reply:
thanks @dr-shorthair - this will require changes in the spec, @andrea-perego @plehegar can we still change this on this round?

Reply:
I agree with the revisions proposed by @dr-shorthair . My understanding is that they can be considered just an editorial change, so it shouldn't be a problem.

Reply:
done